pkg/printinit/noimport.go

```diff
@@ -0,0 +1,9 @@
+package printinit
+
+// we expect this helper function to be called at very early stage of initialization, because this package has no
+// imports.
+// There are some failing CI indicating it takes at least 10 seconds to start a DM-master, we need to know if the OS
+// spawn processes too slowly or some init() are slow to finish.
+func init() {
+	println("init now")
```

Comment:
print a time.Now here？

Reply:
that will introduce an import, which will lower the probability that this init is at early stage of initialization